### PR TITLE
[frinx-machine-monitoring] Update promtail

### PR DESCRIPTION
**Pull request by JumasJM**
#### Checklist

- [ ] Title of the PR starts with chart name (e.g. `[krakend]`)
- [ ] Update the documentation if a new value has been added
- [ ] Update chart release notes *[annotation](https://artifacthub.io/docs/topics/annotations/helm/)*

### CODE REVIEW

File: charts/frinx-machine-monitoring/values.yaml [hunk 1]
```diff
@@ -86,7 +86,19 @@ promtail:
...
     snippets:
       pipelineStages:
-        - cri: {}
+        - match:
+            selector: '{container="uniconfig"}'
+            stages:
+              - cri: {}
+              - multiline:
+                  # Identify timestamps as first line of a multiline block.
+                  firstline: '^\d{1,2}:\d{2}:\d{2}.\d{3}'
```
Comment (Jozefiel) [inline charts/frinx-machine-monitoring/values.yaml:95]: firstline: '^\d{4}-\d{1,2}-\d{1,2} \d{1,2}:\d{2}:\d{2}.\d{3}'

File: charts/frinx-machine-monitoring/values.yaml [hunk 2]
```diff
@@ -86,7 +86,19 @@ promtail:
       - url: http://loki:3100/loki/api/v1/push
     snippets:
       pipelineStages:
-        - cri: {}
+        - match:
+            selector: '{container="uniconfig"}'
+            stages:
+              - cri: {}
```
Comment (Jozefiel) [inline charts/frinx-machine-monitoring/values.yaml:92]: - cri: {}
                  - docker: {}

File: charts/frinx-machine-monitoring/values.yaml (hunk 1 above)
Comment (JumasJM) [reply charts/frinx-machine-monitoring/values.yaml]: resolved

File: charts/frinx-machine-monitoring/values.yaml (hunk 2 above)
Comment (JumasJM) [reply charts/frinx-machine-monitoring/values.yaml:92]: resolved

**Review by Jozefiel — APPROVED** [review]

**Pull request merged**